digid_eherkenning/models/digid.py

```diff
@@ -36,6 +37,17 @@ class DigidConfiguration(BaseConfiguration):
         help_text=_("If enabled, Single Logout is supported"),
     )
 
+    artifact_resolve_content_type = models.CharField(
+        _("resolve artifact binding content type"),
+        choices=XMLContentTypes.choices,
+        default=XMLContentTypes.soap_xml,
+        max_length=100,
+        help_text=_(
+            "'application/soap+xml' is considered legacy and modern brokers typically "
+            "expect 'text/xml'."
+        ),
+    )
```

Comment:
This looks the same like in eHerkenning config, shouldn't we move that field to the abstract base model `BaseConfiguration` instead?

Reply:
It's exactly the same. I think we can move the processing logic to `BaseSaml2Client` too. It's tied to SAML